docs_src/microservices/general/messagebus.md

```diff
@@ -22,14 +22,13 @@ The EdgeX services intended as external entry points are:
 
 - **Core Command External MQTT Connection** - Core Command now receives command requests and publishes responses via an external MQTT connection that is separate from the EdgeX MessageBus. The requests are forwarded to the EdgeX MessageBus and the corresponding responses are forwarded back to the external MQTT connection. 
 
-Originally the EdgeX MessageBus was only used to send *Event/Readings* from Core Data to the Application Services layer. In recent V2 releases more services are now using the EdgeX MessageBus rather than REST for inner service communication. 
+Originally the EdgeX MessageBus was only used to send *Event/Readings* from Core Data to the Application Services layer. In recent releases more services are now using the EdgeX MessageBus rather than REST for inner service communication. 

```

Comment:
```suggestion
Originally, the EdgeX MessageBus only sent *Event/Readings* from Core Data to the Application Services layer. In recent releases, more services use the EdgeX MessageBus rather than REST for inter service communication. 

```

Reply:
fix, with tweaks.
https://github.com/edgexfoundry/edgex-docs/pull/1226